Bio/Data/PDBData.py

```diff
@@ -15,13 +15,21 @@
 # The 'fmt:off' lines prevent black for formatting the dictionaries.
 
 from Bio.Data.IUPACData import protein_letters_3to1 as _protein_letters_3to1
+from Bio.Data.IUPACData import (
+    protein_letters_3to1_extended as _protein_letters_3to1_extended,
+)
 
 # Protein
 protein_letters_3to1 = {k.upper(): v for k, v in _protein_letters_3to1.items()}
 protein_letters_1to3 = {v: k for k, v in protein_letters_3to1.items()}
+# Add the extended amino acid one letter codes from IUPACData.py
+protein_letters_3to1_extended_one_letter = {
+    k.upper(): v for k, v in _protein_letters_3to1_extended.items()
+}
 
 # fmt: off
 protein_letters_3to1_extended = {
+    **protein_letters_3to1_extended_one_letter,
```

Comment:
Refer to my last reply in the general comment thread as to why I don't think this is necessarily appropriate. Some of the entries in `IUPACData` will _not_ be present in PDB files, and as such should not be included here. I'd rather this dictionary be updated manually rather than a bulk update.